package.json

```diff
@@ -9,16 +9,15 @@
   "version": "0.1.0",
   "description": "Tests for Azure REST API Specifications",
   "license": "MIT",
-  "dependencies": {
+  "devDependencies": {
+    "mocha": "*",
     "async": "^2.1.4",
     "glob": "^5.0.14",
     "json-schema-ref-parser": "^3.1.2",
     "lodash": "^3.10.1",
     "request": "^2.61.0",
-    "z-schema": "^3.16.1"
-  },
-  "devDependencies": {
-    "mocha": "*"
+    "z-schema": "^3.16.1",
+    "openapi-validation-tools": "amarzavery/openapi-validation-tools#sway4"
```

Comment:
Curious: Why are we using `amarzavery` fork of `openapi-validation-tools` of sway?